language-support/hs/bindings/src/DA/Ledger/Services/TransactionService.hs

```diff
@@ -0,0 +1,72 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module DA.Ledger.Services.TransactionService (
+    ledgerEnd,
+    GetTransactionsRequest(..), getTransactions,
+    ) where
+
+import Com.Digitalasset.Ledger.Api.V1.LedgerOffset
```

Comment:
Do we really want Java style naming? Or is this inevitable with autogen stuff?

Reply:
These are the names of the services as defined in the .proto files.
For code discoverability, it's useful to keep the direct name correspondence.